tasks/build.py

```diff
@@ -507,49 +558,50 @@ def submit_job(job, submitted_jobs, build_env_cfg, ym, pr_id):
 
     # sbatch output is 'Submitted batch job JOBID'
     #   parse job id & add it to array of submitted jobs PLUS create a symlink from main pr_<ID> dir to job dir (job[0])
-    log(f'submit_build_jobs(): sbatch out: {cmdline_output}')
-    log(f'submit_build_jobs(): sbatch err: {cmdline_error}')
+    log(f"{fn}(): sbatch out: {cmdline_output}")
+    log(f"{fn}(): sbatch err: {cmdline_error}")
 
     job_id = cmdline_output.split()[3]
-    submitted_jobs.append(job_id)
-    symlink = os.path.join(build_env_cfg[JOBS_BASE_DIR], ym, pr_id, job_id)
-    log(f"jobs_base_dir: {build_env_cfg[JOBS_BASE_DIR]}, ym: {ym}, pr_id: {pr_id}, job_id: {job_id}")
+    ym = job.year_month
+    pr_id = job.pr_id
```

Comment:
This saves 3 characters, seems overkill, just use `job.pr_id`?

Also use `job.year_month` rather than the cryptic `ym`

Reply:
changed